.github/workflows/ci.yml

```diff
@@ -196,3 +207,19 @@ jobs:
               if [ $? -ne 0 ]; then exit 1; fi; \
           fi;
           make realclean
+      - name: Test Jacobian
```

Comment:
I am going to leave this to Jon to comment on this. I love the idea though! Also, it looks like there is a `make TPL` there. It feels like maybe that can be reused? I don't know how though...

Reply:
Yeah let's only do `make TPL` in the dependencies step. Looks possible.

Reply:
OK, I think I should also move the `poetry install` there